typing/typecore.ml

```diff
@@ -382,10 +382,15 @@ let unify_pat_types_return_equated_pairs ?(refine = None) loc env ty ty' =
 let unify_pat_types ?refine loc env ty ty' =
   ignore (unify_pat_types_return_equated_pairs ?refine loc env ty ty')
 
-let unify_pat ?refine env pat expected_ty =
+
+(* Unify expected and actual pattern type.
+   [sdesc_for_hint] is used to show integer literals in their original
+   formatting in the "Did you mean ...?" hint, see
+   [report_literal_type_constraint]. *)
+let unify_pat ?refine ?sdesc_for_hint env pat expected_ty =
```

Comment:
The term `actual` type is inexact. I would suggest that you only document the argument that you added.
Moreover, it might be better to document the intent rather than the precise use case:
```ocaml
(** [sdesc_for_hint] is used by error messages to report literals in their original formatting *)
```

Reply:
Thanks for the suggestion! I've adopted it wholesale.